app/controllers/api/users_controller.rb

```diff
@@ -26,5 +26,23 @@ def login
         }
       }.to_json
     end
+
+    def index
```

Comment:
As a standard,
- `index` represents a request for a **list of entities**
- `show` represents a request for only **one entity**

This request returns only one user.